elements/bulbs-video/components/revealed.js

```diff
@@ -202,7 +202,7 @@ export default class Revealed extends React.Component {
   }
 
   makeVideoPlayer (element, videoMeta) {
-    element.id = jwPlayerIdCounter++;
+    element.id = `jw-player-${jwPlayerIdCounter++}`;
```

Comment:
tested it locally with the onion homepage player and it does, not sure if that will change with multiple players, I'll ping you when the link is up